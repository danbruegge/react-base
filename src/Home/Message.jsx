import styled from 'styled-components';

export default styled.p`
    color: DimGrey;
    font-size: 25px;
    margin: 0;
    padding: 5px;
`;
