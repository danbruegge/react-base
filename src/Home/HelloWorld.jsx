import React from 'react';

import Title from './Title';
import Message from './Message';

export default () => (
    <div>
        <Title>Hello World</Title>
        <Message>This is a hello world message.</Message>
    </div>
);
