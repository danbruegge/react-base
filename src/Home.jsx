import { injectGlobal } from 'styled-components';

import HelloWorld from './HelloWorld';

injectGlobal`
    body {
        display: flex;
        justify-content: center;
    }
`;

export default HelloWorld;
