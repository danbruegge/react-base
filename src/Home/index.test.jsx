import React from 'react';
import renderer from 'react-test-renderer';

import Home from 'Home';

describe('Home', () => {
    test('should render component', () => {
        const component = <Home />;
        const tree = renderer.create(component).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
