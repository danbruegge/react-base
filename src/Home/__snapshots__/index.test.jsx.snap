// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render component 1`] = `
<div>
  <h1
    className="sc-bdVaJa hitZWa"
  >
    Hello World
  </h1>
  <p
    className="sc-bwzfXH hRnPxo"
  >
    This is a hello world message.
  </p>
</div>
`;
